sdks/python/apache_beam/dataframe/transforms.py

```diff
@@ -45,6 +45,7 @@
 MAX_PARTITIONS = 1000
 DEFAULT_PARTITIONS = 100
 MIN_PARTITIONS = 10
+PER_COL_OVERHEAD = 1000
```

Comment:
I assume this was determined empirically?

Reply:
Yeah. To within an order of magnitude at least, rounded up.